test/modules/polyfills.js

```diff
@@ -1,6 +1,9 @@
 /* global QUnit, $, testLog */
 
 ( function() {
+    QUnit.testStart((details) => {
```

Comment:
Why is this here as opposed to somewhere else? Because this is the first test file run? May be worth adding a comment to that effect.

Reply:
Or would it make sense to put it in the helpers file?

Reply:
Yes, it was just because that was the first of the test files that got called. I moved it to `test.js` immediately above where the `$.console` gets disabled, which seems like a more logical place.